nightfall-optimist/src/workers/transaction-submitted-app.mjs

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Sure, I'm not saying it is totally bad, but I'm just following community recommendations. What if we clusterized the Optimist instead of just the "receiving transaction" part? For me makes more sense, however we might have issues with WebSockets.